fuseclient.sh

```diff
@@ -3,11 +3,43 @@ JAR_NAME="ega-fuse-0.0.1-SNAPSHOT.jar"
 PATH_TO_JAR="target/$JAR_NAME"
 PROCESSCNT=$(ps x |grep -v grep |grep -c "$JAR_NAME")
 PID=$(ps aux | grep "$JAR_NAME" | grep -v grep | awk '{print $2}')
+ARGS=$2
```

Comment:
So the shell script only supports the "-cf" argument and it doesn't support the others (e.g. "-m", "-c"), does it?

Reply:
Hi @bela-juhasz, the other args like -m, -c etc are still supported. Here the change is only like it will prompt for username & password only when -cf is not available in the command line Arguments.